proposals/0025-full-length-post-content-in-rss-feeds.md

```diff
@@ -0,0 +1,73 @@
+- Start Date: 2022-08-18
+- Reference Issues: <!-- related issues, otherwise leave empty -->
+- Implementation PR: <!-- leave empty -->
+
+# Summary
+
+Updating @astrojs/rss to generate feeds that include full `compiledContext()` strings as `content:encoded` in the XML, given a parameter of `contentLength` with a value of `full`.  (This value would default to `summary`, which is the current default behavior.)
```

Comment:
Should this be `compiledContent` instead?

Reply:
Good catch.